README.md

```diff
@@ -14,13 +14,81 @@ the AGW implements an evolved packet core (EPC), and a combination of an AAA and
 with existing, unmodified commercial radio hardware.<br>
 For more information on Magma please visit the [official website](https://magmacore.org/).
 
+> **WARNING:** Installing this snap will affect your computer's networking configuration.
+> Make sure it is installed on designated hardware (personal computers are strongly discouraged).
+
+### System requirements
+
+**Hardware (baremetal strongly recommended)**
+
+- Processor: x86-64 dual-core processor (around 2GHz clock speed or faster)
+- Memory: 4GB RAM
+- Storage: 32GB or greater SSD
+
+**Networking**
+
+- At least two ethernet interfaces (SGi and S1)
+- Internet connectivity from SGi interface
+
+**Operating System**
+
+- Ubuntu 20.04 LTS
+  ([Ubuntu installation guide](https://help.ubuntu.com/lts/installation-guide/amd64/index.html))
+
+
 ## Usage
 
-Please see
-[How-to: Deploy Magma Access Gateway using snap](/documentation/how-to-guides/deploy_magma_access_gateway.md).
+> **NOTE**: Unless specified otherwise, all actions in this guide are executed using `root`
+> account.
+
+### 1. Install
+
+> **NOTES:**<br>
+> 1. To see the list of currently supported configuration options, execute:
+> ```bash
+> magma-access-gateway.install --help
+> ```
+> 2. Installation progress can be monitored at all times using `journalctl`.
```

Comment:
Note 2 can be removed if you agree with my recommendation below. `journalctl` is absolutely needed so I wouldn't put it as a note.

Reply:
In the end, we would only have the content of note 3 here.

Reply:
Please see my comment below - in general I agree, but at this point, we can't eliminate `journalctl` as it's the only place which will have all the logs.
I'll remove this note in the [new version](https://github.com/canonical/magma-access-gateway-snap/pull/14) in which we can easily print everything to console.